docs/content/adoption/companies_experience.md

```diff
@@ -0,0 +1,32 @@
+# Companies experience
+
+Looking to how other companies implemented `UI/Snapshot testing` may help you to choose proper tools for your project
+and save your time.
+
+We would be really grateful if you
+could [contribute and share your experience](https://android-ui-testing.github.io/Cookbook/home/contribution) page to
+help other people.
+
+## Revolut
+
+`UI testing`
+
+:   
+`Write:` [Kaspresso](https://github.com/KasperskyLab/Kaspresso) <br>
+`Who:` Android Engineers<br>
+`Runner:` [Marathon](https://android-ui-testing.github.io/Cookbook/practices/test_runners_review/#21-marathon) <br>
+`Where:` [Headless emulators in Docker (Avito Image)](https://hub.docker.com/r/avitotech/android-emulator-29) <br>
+`How often:` Each 4h and before each release <br>
+`Network:` Mock, by Custom [OkReplay](https://github.com/airbnb/okreplay) <br>
+`Test report:` [Allure](http://allure.qatools.ru/)<br>
+
+`Snapshot testing`
+
```

Comment:
I did not know you were using Screenshot tests at Revolut as well
![](https://media.giphy.com/media/3ohzdIuqJoo8QdKlnW/giphy-downsized.gif)